src/Symfony/Bundle/TwigBundle/Resources/views/Exception/exception_full.html.twig

```diff
@@ -134,7 +134,7 @@
 {% endblock %}
 
 {% block title %}
-    {{ exception.message }} ({{ status_code }} {{ status_text }})
+    {{ exception.message }} (HTTP status code: {{ status_code }} - {{ status_text }})
```

Comment:
does this really improve the page?

Reply:
For new member of our community title `(404 - Not Found)` or `(500 - Internal Server Error)` may sound strange. This number looks like Sf internal error number, and I think that it can help with understanding this. It's only my opinion ;)